hathor/transaction/static_metadata.py

```diff
@@ -68,7 +62,6 @@ def from_bytes(cls, data: bytes, *, target: 'BaseTransaction') -> 'VertexStaticM
         raise NotImplementedError
 
 
-@dataclass(slots=True, frozen=True, kw_only=True)
```

Comment:
Why are you removing this line?

Reply:
I mentioned it in the Acceptance Criteria:

> Change static metadata dataclasses to `BaseModel`s instead, which will be necessary for (de)serialization.

Specifically, I'm now serializing the `Feature` and `FeatureState` enums. Before, I was using `dataclasses.asdict()` which serializes enums as objects that cannot be converted to json, instead of using their raw values directly. Then, I changed those classes to `BaseModel` which are equivalent and provide the `json_dumpb()` method, so I don't have to do this conversion manually.

Reply:
Is `BaseModel` mutable? If yes, I don't think it's worth losing immutability because of a serialization method.

Reply:
It's immutable, as configured here: https://github.com/HathorNetwork/hathor-core/blob/da32f8092a1ae038046864a4edc6e448b9e1b811/hathor/utils/pydantic.py#L33

Reply:
Got it. Thanks for clarifying.